src/Minsk/CodeAnalysis/Compilation.cs

```diff
@@ -61,18 +61,18 @@ public IEnumerable<Symbol> GetSymbols()
             var submission = this;
             var seenSymbolNames = new HashSet<string>();
 
+            const ReflectionBindingFlags bindingFlags =
```

Comment:
We should replace all of this with [BuiltinFunctions.GetAll()](https://github.com/terrajobst/minsk/blob/master/src/Minsk/CodeAnalysis/Symbols/BuiltinFunctions.cs#L15).

Reply:
Even better :-)